pom.xml

```diff
@@ -898,6 +898,7 @@
               <exclude>**/interpreter-setting.json</exclude>
               <exclude>**/constants.json</exclude>
               <exclude>scripts/**</exclude>
+              <exclude>**/src/main/java/org/apache/zeppelin/jdbc/SqlCompleter.java</exclude>
```

Comment:
shall we add a comment here why this is excluded? or is there another way to track the source of given file?